test/sharness/t0160-resolve.sh

```diff
@@ -0,0 +1,72 @@
+#!/bin/sh
+
+test_description="Test resolve command"
+
+. lib/test-lib.sh
+
+test_init_ipfs
+
+test_expect_success "resolve: prepare files" '
+	mkdir -p a/b &&
+	echo "a/b/c" >a/b/c &&
+	a_hash=$(ipfs add -q -r a | tail -n1) &&
+	b_hash=$(ipfs add -q -r a/b | tail -n1) &&
+	c_hash=$(ipfs add -q -r a/b/c | tail -n1)
+'
+
+test_resolve_setup_name() {
+	ref=$1
+
+	test_expect_success "resolve: prepare name" "
```

Comment:
I am not sure it would work if it is single quoted. In some cases I think it is ok to double quote the second argument.